ci-operator/step-registry/ipi/conf/user-workload-monitoring/ipi-conf-user-workload-monitoring-commands.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# Use yq to create cluster monitoring config, as other steps may adjust it
+curl -L https://github.com/mikefarah/yq/releases/download/3.3.0/yq_linux_amd64 -o /tmp/yq && chmod +x /tmp/yq
+
+CONFIG="${SHARED_DIR}/manifest_cluster-monitoring-config.yaml"
+PATCH="${SHARED_DIR}/cluster-monitoring-config.yaml.patch"
```

Comment:
nit: seems like holding the patch in the shared directory is traditional, but I don't see why we'd ever want to share patches.  I've filed #20865 to fix our existing use-cases, so you should be able to fix this without worrying about diverging from the pattern ;).

Reply:
Good idea, fixed